app/chat/[id].tsx

```
@@ -169,6 +188,7 @@ export default function Chat() {
                     <PromptInput input={input}
                                  setInput={setInput}
                                  handleSubmit={handleSubmit}
+                                 chatAgentMode={chatAgentMode!}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)